piet-common/Cargo.toml

```diff
@@ -14,34 +14,36 @@ cairo = ["piet-cairo", "cairo-rs"]
 web = ["piet-web"]
 
 [dependencies]
-cfg-if = "0.1.10"
 piet = { version = "0.1.0", path = "../piet" }
 piet-cairo = { version = "0.1.0", path = "../piet-cairo", optional = true }
 piet-coregraphics = { version = "0.1.0", path = "../piet-coregraphics", optional = true }
 piet-direct2d = { version = "0.1.0", path = "../piet-direct2d", optional = true }
 piet-web = { version = "0.1.0", path = "../piet-web", optional = true }
-cairo-rs = { version = "0.8.1", default_features = false, optional = true}
 
-[target.'cfg(not(any(target_arch="wasm32", target_os="windows", target_os="macos")))'.dependencies]
+cfg-if = "0.1.10"
+cairo-rs = { version = "0.8.1", default_features = false, optional = true }
+png = { version = "0.16.1", optional = true }
+
+[target.'cfg(target_os="linux")'.dependencies]
 piet-cairo = { version = "0.1.0", path = "../piet-cairo" }
 cairo-rs = { version = "0.8.1", default_features = false}
```

Comment:
will these get added to a wasm32 build on linux? or is that not a thing?

Reply:
They will not be compiled. There are no warnings or anything either, so I think it's fine to not make the cfg guard more complex here.